app/views/plants/_add_plant_img.html.haml

```diff
@@ -0,0 +1,4 @@
+= form_for @plant do |f|
+  = f.label :img
+  = f.file_field :img
+  = f.submit 'Change picture', class: 'btn-primary mtn-2'
```

Comment:
We currently have a picture uploader function with motor_admin, I don't think we should open this particular function to all users.

Reply:
Yes the partial reference is commented out in the show page.

Reply:
I understand you, but we should ensure to keep the codebase clutter free, if the code snippet is not being used, it should be removed, just commenting it out should be discouraged.